src/pybind/mgr/mgr_module.py

```diff
@@ -59,22 +59,41 @@ def emit(self, record):
         self._module._ceph_log(ceph_level, self.format(record))
 
 
-def configure_logger(module_inst, name):
-    logger = logging.getLogger(name)
+def configure_logger(module_inst, module_name):
+    """
+    Create and configure the logger with the specified module.
+
+    A handler will be added to the root logger which will redirect the
+    messages from all loggers (incl. 3rd party libraries) to the Ceph log.
 
-    # Don't filter any logs at the python level, leave it to C++
-    logger.setLevel(logging.DEBUG)
+    :param module_inst: The module instance.
+    :type module_inst: instance
+    :param module_name: The module name.
+    :type module_name: str
+    :return: Return the logger with the specified name.
+    """
+    logger = logging.getLogger(module_name)
+    # Don't filter any logs at the python level, leave it to C++.
+    logger.setLevel(logging.NOTSET)
 
     # FIXME: we should learn the log level from C++ land, and then
     # avoid calling the C++ level log when we know a message is of
     # an insufficient level to be ultimately output
-    logger.addHandler(CPlusPlusHandler(module_inst))
+
+    # Add handler to the root logger, thus this module and all
+    # 3rd party libraries will log their messages to the Ceph log.
+    root_logger = logging.getLogger()
```

Comment:
Does it make sense to set the level of `root_logger` to ERROR?

Reply:
Done, but keep in mind how the effective log level is determined. If a 3rd party library set a other level than ``ogging.NOTSET``, then this level will be used and no further filtering in ancestor loggers is done.

Reply:
was meant to just be a default for 3rd party libraries. for example, the `requests` library is logging too much on the INFO level.